controllers/apps/apimanager_controller.go

```diff
@@ -363,6 +380,11 @@ func (r *APIManagerReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			handler.EnqueueRequestsFromMapFunc(configMapToApimanagerEventMapper.Map),
 			builder.WithPredicates(resourceVersionChangePredicate),
 		).
+		Watches(
```

Comment:
@eguzki , existing mapper `configMapToApimanagerEventMapper` was introduced just few days before by @MStokluska , but I tried it now, and it's not working for me - it's not restoring `redis-config` CM contents in case of change of the configmap. Maybe we can use my APIManagerConfigMapsEventMapper meanwhile, but we can open follow-on ticket for 2.16 release to "merge"  APIManagerConfigMapsEventMapper  and ConfigMapToApimanagerEventMapper  ? Or you think need to do  it now?

Reply:
Done. Using Owns. APIManagerConfigMapsEventMapper - removed
Retested
Thank yhou